### ISSUE TITLE
Use custom Bootstrap checkbox

### ISSUE THREAD

**Issue by SamThilmany** [davidstutz/bootstrap-multiselect#992]
The documentation is quite unclear in explaining the templates, that's why I'm asking this as a question.

How can I replace the default checkbox that is set by the `bootstrap-multiselect` plugin by the custom checkbox defined in [Bootstrap 4](https://getbootstrap.com/docs/4.0/components/forms/#custom-forms)?

Having more flexibility in using the plugin would really be great. Please freshen up the documentation, especially the templates part.

**Comment by adryx92** [comment]
It seems not to be compatible with Bootstrap 4.

**Comment by SamThilmany** [comment]
@adryx92 Yeah it's only stated to be compatible with BS 3 but even with the custom forms of BS 3, I cannot see a method to get this working.
